leonardo-example.conf

```diff
@@ -39,6 +39,11 @@ dataproc {
   # If the subnet is in a shared VPC, this field can accept a partial URI (ex.
   # "projects/[projectId]/regions/us-east1/sub0") in addition to the short-name form "sub0".
   #vpcSubnet = "default"
+
+  # Optional. The custom dataproc image to use. If not specified Leo will choose a public Dataproc
+  # image appropriate for the docker image being installed.
+  # See: https://cloud.google.com/dataproc/docs/guides/dataproc-images
+  #customDataprocImage = "projects/[project_id]/global/images/[image_name]"
```

Comment:
should this be in reference.conf with a manually generated image?

Reply:
Yeah... I'm also wondering if this should be a required param. It could default to `1.2-deb9` in reference.conf, and be overridden with a custom-image in leonardo.conf.

Reply:
yea, I was wondering if there's anything else we're waiting on. Like can we use the custom image in reference.conf today? Even if not, can we make the parameter `dataprocImage` and use the default image for now (as apposed to use an optional parameter?

Reply:
I think there's a reason I didn't want to do this.. let's chat in person. :)

Reply:
Facetalked - i specified the custom image in reference.conf, will watch the tests!